docs/executor/kubernetes.rst

```diff
@@ -41,6 +41,62 @@ The volumes are optional and depend on your configuration. There are two volumes
 To troubleshoot issue with KubernetesExecutor, you can use ``airflow kubernetes generate-dag-yaml`` command.
 This command generates the pods as they will be launched in Kubernetes and dumps them into yaml files for you to inspect.
 
+.. _concepts:pod_template_file:
+
+pod_template_file
+#################
```

Comment:
Needs a rst style heading

Reply:
@kaxil I'm not sure what you mean? This seems to be the same as the sections below it?

Reply:
aah it works, yeah ignore me